### PR TITLE
Royd/multisig register push

### PR DESCRIPTION
**Pull request by royd**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -444,6 +444,54 @@ class _ProvenanceWalletAppState extends State<ProvenanceWalletApp> {
...
+        }
+      }
+    }).addTo(_subscriptions);
+
+    accountService.events.updated.listen((e) {
+      final address = e.publicKey?.address;
+      if (address != null && !remoteNotificationService.isRegistered(address)) {
+        remoteNotificationService.registerForPushNotifications(address).onError(
+              (error, stackTrace) => logDebug(
+                'Failed to register for push notifications for account: $address',
```
Comment (jakegraves-figure) [inline lib/main.dart:490]: This message and the first one are the same. Consider putting something like `'Failed to update push notifications for account: $address'` so we have a better idea where to look if there is some kind of issue.

File: lib/screens/home/transactions/transaction_details_screen.dart [hunk 2]
```diff
@@ -122,7 +122,7 @@ class TransactionDetailsScreen extends StatelessWidget {
                     onTap: () async {
                       final account = await accountService.getSelectedAccount();
                       String url;
-                      switch (account?.publicKey?.coin) {
+                      switch (account?.publicKey.coin) {
```
Comment (jakegraves-figure) [inline lib/screens/home/transactions/transaction_details_screen.dart:125]: Yay!

**Review by jakegraves-figure — DISMISSED** [review]

**Review by dhancockFigure — APPROVED** [review]

File: lib/main.dart (hunk 1 above)
Comment (royd) [reply lib/main.dart:490]: good point

**Comment by royd** [comment]
Moving to a new branch

**Pull request closed without merging**